### ISSUE TITLE
'train_acc': -1,   'valid_acc': -1

### ISSUE THREAD

**Issue by songziyu2333** [autogluon/autogluon#2158]
The following error occurred when I was using the official documentation tutorial：

env： torch==1.12.0+cu113 torchvision==0.13.0+cu113 torchtext==0.13.0  as same as official documentation  windos+python3.8.5

`time_limit=auto` set to `time_limit=7200`.
Reset labels to [0, 1, 2, 3]
Randomly split train_data into train[720]/validation[80] splits.
The number of requested GPUs is greater than the number of available GPUs.Reduce the number to 1
Starting fit without HPO
modified configs(<old> != <new>): {
root.img_cls.model   resnet101 != resnet50
root.train.epochs    200 != 50
root.train.early_stop_baseline 0.0 != -inf
root.train.batch_size 32 != 16
root.train.early_stop_max_value 1.0 != inf
root.train.early_stop_patience -1 != 10
root.misc.seed       42 != 428
root.misc.num_workers 4 != 12
}
Saved config to C:\Users\HP\Desktop\autogloun\2b000416\.trial_0\config.yaml
Model resnet50 created, param count:                                         23516228
AMP not enabled. Training in float32.
Disable EMA as it is not supported for now.
Start training from [Epoch 0]
`time_limit=auto` set to `time_limit=7200`.
Reset labels to [0, 1, 2, 3]
Randomly split train_data into train[720]/validation[80] splits.
The number of requested GPUs is greater than the number of available GPUs.Reduce the number to 1
Starting fit without HPO
modified configs(<old> != <new>): {
root.img_cls.model   resnet101 != resnet50
root.misc.num_workers 4 != 12
root.misc.seed       42 != 204
root.train.early_stop_patience -1 != 10
data/
├── test/
└── train/
root.train.early_stop_max_value 1.0 != inf
root.train.batch_size 32 != 16
root.train.early_stop_baseline 0.0 != -inf
root.train.epochs    200 != 50
}
Saved config to C:\Users\HP\Desktop\autogloun\1444b6bf\.trial_0\config.yaml
Model resnet50 created, param count:                                         23516228
AMP not enabled. Training in float32.
Disable EMA as it is not supported for now.
Start training from [Epoch 0]
Finished, total runtime is 1.50 s
{ 'best_config': { 'batch_size': 16,
                   'dist_ip_addrs': None,
                   'early_stop_baseline': -inf,
                   'early_stop_max_value': inf,
                   'early_stop_patience': 10,
                   'epochs': 50,
                   'final_fit': False,
                   'gpus': [0],
                   'lr': 0.01,
                   'model': 'resnet50',
                   'ngpus_per_trial': 8,
                   'nthreads_per_trial': 128,
                   'num_workers': 12,
                   'searcher': 'random',
                   'seed': 204,
                   'time_limits': 7200},
  'total_time': 1.4981746673583984,
  'train_acc': -1,
  'valid_acc': -1}

**Comment by bryanyzhu** [comment]
Can I ask which tutorial you had run?

**Comment by songziyu2333** [comment]
> Can I ask which tutorial you had run?  
sure, it is Image Prediction , I was stuck by  the  quick start . And here is the code
import autogluon.core as ag
from autogluon.vision import ImagePredictor, ImageDataset
train_dataset, _, test_dataset = ImageDataset.from_folders('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
print(train_dataset)
predictor = ImagePredictor()
# since the original dataset does not provide validation split, the `fit` function splits it randomly with 90/10 ratio
predictor.fit(train_dataset, hyperparameters={'epochs': 2})  # you can trust the default config, we reduce the # epoch to save some build time

and then report error

**Comment by songziyu2333** [comment]
> Can I ask which tutorial you had run?

sure, it is Image Prediction , I was stuck by the quick start . And here is the code
import autogluon.core as ag
from autogluon.vision import ImagePredictor, ImageDataset
train_dataset, _, test_dataset = ImageDataset.from_folders('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
print(train_dataset)
predictor = ImagePredictor()

since the original dataset does not provide validation split, the fit function splits it randomly with 90/10 ratio
predictor.fit(train_dataset, hyperparameters={'epochs': 2}) # you can trust the default config, we reduce the # epoch to save some build time

and then report error

**Comment by bryanyzhu** [comment]
I have run the tutorials, and can't reproduce this error. 

I also found your output a bit strange, e.g., (1) if the epochs is set to 2, why your best_config shows `'epochs': 50,`? (2) Normally training would take some time, like 10+ seconds, but yours only take 1.5 seconds. It seems the model is not trained at all. That's why the low accuracy. 

It might because of the windows environment, a good way to test is to open a colab in browser and test the code.

**Comment by songziyu2333** [comment]
> I have run the tutorials, and can't reproduce this error.
> 
> I also found your output a bit strange, e.g., (1) if the epochs is set to 2, why your best_config shows `'epochs': 50,`? (2) Normally training would take some time, like 10+ seconds, but yours only take 1.5 seconds. It seems the model is not trained at all. That's why the low accuracy.
> 
> It might because of the windows environment, a good way to test is to open a colab in browser and test the code.

thank u for helping, for the first question, it is because I set epoch 50, and when it set epo 2 also has this problem
and for the second question, i think it is better to solve this problem not avoid it, maybe I will test on colab, but I still don't know what happend. thanks again.

**Comment by bryanyzhu** [comment]
I agree, please let me know how it goes on colab. I can also locate a windows environment later to look into this issue.

**Comment by songziyu2333** [comment]
thank u , I can successfully run the tutorial on colab

**Comment by Innixma** [comment]
Marking as resolved, use autogluon.multimodal for vision tasks going forward in v0.6 and v0.7+
